packages/backend/server/src/plugins/copilot/context/session.ts

```diff
@@ -164,7 +164,10 @@ export class ContextSession implements AsyncDisposable {
   }
 
   async removeFile(fileId: string): Promise<boolean> {
-    await this.models.copilotContext.deleteEmbedding(this.contextId, fileId);
+    await this.models.copilotContext.deleteFileEmbedding(
+      this.contextId,
+      fileId
+    );
```

Comment:
Where is the cleanup logic for additional files uploaded by users in the worksapce setting?

Reply:
removeWorkspaceEmbeddingFiles